csrc/transform_iter.cpp

```diff
@@ -253,7 +262,7 @@ void ReplayTransformations::handle(Resize* exp) {
         mapped,
         exp->leftExpand(),
         exp->rightExpand(),
-        mapped->isRFactorProduct());
```

Comment:
I suspect this should have been `false` always. It would have resulted in an `rfactor` domain outside of the root-rfactor transformation.

Reply:
Agreed. I did try to test this line of change alone and found all tests passed. Probably a lack of test coverage